### ISSUE TITLE
{Bug} The furnace does not proceed to explode if there is an item in the output slot.

### ISSUE THREAD

**Issue by ghost** [esWhistler/DangerousGunpowderFabric#4]
If you have been smelting something else in the furnace before putting in the gunpowder and have not taken out the output item before putting the gunpowder in the furnace will never explode (until you take out the output item)

**Comment by esWhistler** [comment]
Yeah, this happens because of the way the mod works, it's a actually a crafting recipe that creates a dummy item and when the game detects that's on the output it creates the explosion. So it behaves like a regular crafting recipe otherwise, I'll probably look into using another method to be able to implement some of your suggestions anyway so I'll try to make it so this also doesn't happen
